src/domain/app/sessionPrompt/SessionPrompt.tsx

```diff
@@ -8,17 +8,30 @@ import { closeExpiredSessionPrompt } from '../state/ui/UIActions';
 import Button from '../../../common/components/button/Button';
 import { logoutTunnistamo } from '../../auth/authenticate';
 import { resetBackendAuthentication } from '../../auth/state/BackendAuthenticationActions';
+import { clearProfile } from '../../profile/state/ProfileActions';
+import { persistor } from '../state/AppStore';
+import client from '../../api/client';
 
 const SessionPrompt: React.FunctionComponent<{ isOpen: boolean }> = ({
   isOpen = false,
 }) => {
   const dispatch = useDispatch();
 
   const flushAuthenticationData = () => {
+    // Close prompt
     dispatch(closeExpiredSessionPrompt());
 
+    // Clear api token
     dispatch(resetBackendAuthentication());
 
+    // Clear profile (fetched from API)
```

Comment:
Now we have two places where these logout operations are done, with a small difference. Wonder if it would be better to create a function for this with some parameter? 

A future dev might be confused or annoyed if they for some reason need to add one more reducer (or something) during logout, and then end up adding it two places..